lib/tasks/addresses.rake

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+namespace :address do
+  desc "Fix addresses set in the renewals process to match old data format"
+  task update_from_os_places: :environment do
+    registrations = WasteCarriersEngine::Registration.where(:past_registration.exists => true)
+    transient_registrations = WasteCarriersEngine::TransientRegistration.where(:addresses.exists => true)
+
+    update_addresses_for(registrations)
+    update_addresses_for(transient_registrations)
+  end
+end
+
+def update_addresses_for(registrations)
+  registrations.each do |registration|
+    update_address_if_necessary(registration.registered_address)
+    update_address_if_necessary(registration.contact_address)
+  end
+end
+
+def update_address_if_necessary(address)
```

Comment:
Is there a particular need for this method? Could we not just make it this in `update_addresses_for()`

```ruby
  update_address(address) if address_is_from_os_places?(registration.registered_address)
  update_address(address) if address_is_from_os_places?(registration.contact_address)
```